### PR TITLE
Baskerville Addendums

### PR DESCRIPTION
**Pull request by jnywong**
Baskerville specific addendums to Episodes 11, 12 and 13

### CODE REVIEW

**Review by branfosj — APPROVED** [review]

File: _episodes/11-connecting.md [hunk 1]
```diff
@@ -305,6 +305,12 @@ See the [PuTTY documentation][putty-agent].
 
 ### Transfer Your Public Key
 
+> ## Baskerville
+> 
+> For Baskerville, please paste your **public** SSH key into [Baskerville Admin](https://admin.baskerville.ac.uk/)
+> under the `'Configure account / reset password'` link. and disregard the **s**ecure **c**o**p**y tool below.
```
Comment (JimMadge) [inline _episodes/11-connecting.md]:
```suggestion
> under the `'Configure account / reset password'` link and disregard the **s**ecure **c**o**p**y tool below.
```

File: _episodes/11-connecting.md [hunk 2]
```diff
@@ -327,7 +333,7 @@ supplied by the instructors.
 ```
 {: .language-bash}
 
-You may be asked for your password. Watch out: the characters you type after
+You may be asked for your password, or a One Time Passcode for Multi Factor Authentication. Watch out: the characters you type after
```
Comment (JimMadge) [inline _episodes/11-connecting.md]:
```suggestion
You may be asked for your password, and/or a One Time Passcode for Multi Factor Authentication. Watch out: the characters you type after
```

I think if public-key authentication fails you are prompted for a password then the OTP.

**Review by JimMadge — CHANGES_REQUESTED** [review]
Starting to take shape :+1:.

**Comment by jnywong** [comment]
@branfosj I've included Jim's changes and branch is ready to merge

**Review by JimMadge — APPROVED** [review]

**Pull request merged**